pandas/tests/io/formats/test_to_html.py

```diff
@@ -11,6 +11,22 @@
 from pandas.compat import u, lrange, StringIO
 from pandas.util import testing as tm
 import pandas.io.formats.format as fmt
+import pandas.core.common as com
+
+
+@pytest.fixture(params=[True, False], ids=['index_true', 'index_false'])
```

Comment:
I don't think this warrants a separate fixture; fine to just pass True / False via parametrization to the functions where required